template/lambda-sqs-worker/package.json

```diff
@@ -14,26 +14,30 @@
     "test:watch": "skuba test --watch"
   },
   "dependencies": {
+    "@aws-sdk/client-sns": "^3.213.0",
     "@seek/logger": "^5.0.1",
-    "aws-sdk": "^2.1011.0",
     "datadog-lambda-js": "^6.83.0",
     "dd-trace": "^3.8.0",
     "skuba-dive": "^2.0.0",
     "zod": "^3.19.1"
   },
   "devDependencies": {
+    "@aws-sdk/client-codedeploy": "^3.213.0",
+    "@aws-sdk/client-lambda": "^3.213.0",
```

Comment:
Do we want to be consistent in placing these in `dependencies` vs `devDependencies`?

Reply:
Hmmm these are only needed by the `hook.ts` file. We could in theory rely on the aws-sdk-v3 in the runtime but I do see your point?

Reply:
If we are comfortable relying on the built-in we can move SNS to dev deps too, right?